src/python/zigopt/handlers/training_runs/update.py

```diff
@@ -208,40 +208,3 @@ def handle(self, params):
       checkpoint_count=checkpoint_count,
       project=project,
     )
-
-  """
-    PUT/MERGE /v1/training_runs/[training_run_id]
-    Required permission: client write
-    Description: Update the specified training run.
-      PUT will replace existing values, MERGE will update mappings recursively.
-      If the training run belongs to an experiment and the state is updated to "completed" or "failed"
-      then an observation will be created.
-    Parameters:
-      name: the new name of the training run
-      state: "active", "completed" or "failed"
-      assignments: map of parameter names to their assigned values
-      assignments_meta: map of parameter names to metadata for the associated assignment
-        source: name of the assignment source
-      assignments_sources: map of assignment source names to metadata for the associated source
-        default_show: show or hide the source in the UI by default
-        sort: numerical sort key for the assignment source
-      model: map of information about the model used
-        type: string represenation of the model type
-      source_code: map of information about the source code used
-        content: string contents of source file or notebook cell
-        hash: VCS hash for source code version
-      values: map of metric names to observed metric values
-        value: numerical value of the metric
-        value_stddev: numerical standard deviation of the observed metric value
-      metadata: map for arbitrary key value storage by the user
-      dev_metadata: other metadata for internal use
-      sys_metadata: metadata for integration specific features
-        feature_importances: map of feature importance information
-          scores: map of feature to feature score
-          type: type of feature importance
-    Returns:
-      TrainingRun
-      see zigopt.json.builder.training_run.TrainingRunJsonBuilder
-    Python client:
-      connection.training_runs(training_run_id).update()
-    """
```

Comment:
@Chris-Sigopt I'm not sure what this is doing here. It looks like there is already some description of patch/merge at the top.